package.json

```diff
@@ -12,7 +12,8 @@
     "client:build": "./node_modules/.bin/tsc -p ./src/client/tsconfig.json && ./node_modules/.bin/webpack",
     "seed": "node js/server/seed.js",
     "seed-faker": "node js/server/seed-faker.js",
-    "server:watch": "./node_modules/.bin/tsc -w & ./node_modules/.bin/nodemon --ignore client --ignore src",
+    "seed-faker:dev": "npm run server:build && npm run seed-faker",
+    "server:watch": "./node_modules/.bin/tsc -w & ./node_modules/.bin/nodemon --ignore client --ignore js",
```

Comment:
this should be --ignore client -- ignore src

Reply:
Good catch, i may have not seen this due to a merge conflict